acceptance/lib/puppet/acceptance/common_utils.rb

```diff
@@ -1,24 +1,10 @@
-require 'puppet/acceptance/puppet_helpers'
-extend Puppet::Acceptance::PuppetHelpers
-
 module Puppet
   module Acceptance
-    module CAUtils
```

Comment:
This seems like it was weird before, and continues to be weird after this change.

Before `lib/puppet/acceptance/common_utils.rb` was declaring the namespace `Puppet::Acceptance::CAUtils`, and now it's declaring `Puppet::Acceptance::CommandUtils`.  I'd say that neither of those is correct, and that `lib/puppet/acceptance/common_utils.rb` should be declaring the namespace `Puppet::Acceptance::CommonUtils`